alpaca_trade_api/polygon/rest.py

```diff
@@ -109,27 +110,28 @@ def historic_quotes_v2(
 
         return QuotesV2(raw)
 
-    def historic_agg(self, size, symbol,
-                     _from=None, to=None, limit=None):
-        path = '/historic/agg/{}/{}'.format(size, symbol)
-        params = {}
-        if _from is not None:
-            params['from'] = _from
-        if to is not None:
-            params['to'] = to
-        if limit is not None:
-            params['limit'] = limit
-        raw = self.get(path, params)
-
-        return Aggs(raw)
-
     def historic_agg_v2(self, symbol, multiplier, timespan, _from, to,
                         unadjusted=False, limit=None):
-        path = '/aggs/ticker/{}/range/{}/{}/{}/{}'.format(
-            symbol, multiplier, timespan, _from, to
+        """
+
+        :param symbol:
+        :param multiplier: Size of the timespan multiplier (distance between samples.
+               e.g if 1 we get for daily 2015-01-05, 2015-01-06, 2015-01-07, 2015-01-08
+                   if 3 we get           2015-01-01, 2015-01-04, 2015-01-07, 2015-01-10)
+        :param timespan: Size of the time window: minute, hour, day, week, month, quarter, year
+        :param _from:
+        :param to:
+        :param unadjusted:
+        :param limit: max samples to retrieve (seems like we get "limit - 1" )
+        :return:
+        """
+        path = '/aggs/ticker/{symbol}/range/{multiplier}/{timespan}/{_from}/{to}'.format(symbol=symbol,
```

Comment:
please note that some of our code that calls this method (e.g pylivetrader)
is calling it like this:
```py
self._api.polygon.historic_agg_v2(
                symbol, 1, size,
                int(_from.timestamp()) * 1000,       <====================
                int(to.timestamp()) * 1000      <====================
            )
```
and it doesn't work because this code expects a string.

so we should either fix all calling code to use the same format, or make this code adjustable to this scenario.